html-proofer.gemspec

```
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
   gem.executables   = all_files.grep(%r{^bin/}) { |f| File.basename(f) }
   gem.test_files    = gem.files.grep(%r{^(spec)/})
   gem.require_paths = ['lib']
-  gem.required_ruby_version = '~> 2.4'
+  gem.required_ruby_version = '>= 2.4.10'
```

I'm glad 3.0 passes but this isn't future-proof enough. Could you please change it to `'~> 2.4, <~ 3.0'` (or whatever the syntax for version ranges is, I always forget--basically >= 2.4 but < 4.0).

---

Okay, I'd go with

```suggestion
  gem.required_ruby_version = ['>= 2.4.10', '< 4.0']
```

btw: [Ruby 2.4 is EOL since 2020-04-01](https://www.ruby-lang.org/en/news/2019/04/01/ruby-2-4-6-released/), I'd recommend to drop support soonish.

---

👍